packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -631,6 +635,344 @@ void main() {
+          expect(Map<String, String>.from(jsonDecode(script)), <String, String>{
+            'command': 'get_offset',
+            'offsetType': 'topRight',
+            'timeout': _kSerializedTestTimeout,
+            'finderType': 'ByValueKey',
+            'keyValueString': '123',
+            'keyValueType': 'int',
+          });
+          return jsonEncode(await makeMockResponse(<String, double>{
+            'dx': 11,
+            'dy': 12,
+          }));
+        });
+        final DriverOffset result = await driver.getTopRight(find.byValueKey(123), timeout: _kTestTimeout);
+        expect(result, const DriverOffset(11, 12));
+      });
+
+      test('sends the getBottomLeft command', () async {
+        when(mockConnection.sendCommand(any, any)).thenAnswer((Invocation i) async {
+          final String script = _checkAndEncode(i.positionalArguments[0]);
+          expect(Map<String, String>.from(jsonDecode(script)), <String, String>{
+            'command': 'get_offset',
+            'offsetType': 'bottomLeft',
+            'timeout': _kSerializedTestTimeout,
+            'finderType': 'ByValueKey',
+            'keyValueString': '123',
+            'keyValueType': 'int',
+          });
+          return jsonEncode(await makeMockResponse(<String, double>{
+            'dx': 11,
+            'dy': 12,
+          }));
+        });
+        final DriverOffset result = await driver.getBottomLeft(find.byValueKey(123), timeout: _kTestTimeout);
+        expect(result, const DriverOffset(11, 12));
+      });
+
+      test('sends the getBottomRight command', () async {
+        when(mockConnection.sendCommand(any, any)).thenAnswer((Invocation i) async {
+          final String script = _checkAndEncode(i.positionalArguments[0]);
+          expect(Map<String, String>.from(jsonDecode(script)), <String, String>{
+            'command': 'get_offset',
+            'offsetType': 'bottomRight',
+            'timeout': _kSerializedTestTimeout,
+            'finderType': 'ByValueKey',
+            'keyValueString': '123',
+            'keyValueType': 'int',
+          });
+          return jsonEncode(await makeMockResponse(<String, double>{
+            'dx': 11,
+            'dy': 12,
+          }));
+        });
+        final DriverOffset result = await driver.getBottomRight(find.byValueKey(123), timeout: _kTestTimeout);
+        expect(result, const DriverOffset(11, 12));
+      });
+    });
+
+
+    test('checks the health of the driver extension', () async {
+      when(mockConnection.sendCommand(any, any)).thenAnswer((Invocation i) async {
+        final String script = _checkAndEncode(i.positionalArguments[0]);
+        expect(Map<String, String>.from(jsonDecode(script)), <String, String>{
+          'command': 'get_health',
+        });
+        return jsonEncode(await makeMockResponse(<String, dynamic>{'status': 'ok'}));
+      });
+      await driver.checkHealth();
+    });
+  });
```

Comment:
You should also check the Unimplemented methods, and the chrome only implemenations